Web/HTML/display_articles.html

```diff
@@ -4,219 +4,48 @@
-                                    </li>
-                                    <li>
-                                        <a href="#">Désodorisant</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Renovation et protection</a>
-                                        <ul>
-                                            <li><a href="#">Polish voiture</a></li>
-                                            <li><a href="#">Cire voiture</a></li>
-                                            <li><a href="#">Efface rayures</a></li>
-                                            <li><a href="#">Produit lustrage voiture</a></li>
-                                            <li><a href="#">Rénovateur chrome / alu</a></li>
-                                            <li><a href="#">Kit renovation phare</a></li>
-                                            <li><a href="#">Rénovateur pare-choc</a></li>
-                                            <li><a href="#">Anticorrosif</a></li>
-                                            <li><a href="#">Antirouille</a></li>
-                                            <li><a href="#">Insonorisant</a></li>
-                                            <li><a href="#">Bombe de peinture</a></li>
-                                            <li><a href="#">Autres produits</a></li>
-                                        </ul>
-                                    </li>
-                                    <li>
-                                        <a href="#">Réparation carrossrie</a>
-                                    </li>
-                                </ul>
-                            </li>
+                        <ul id="main-nav2">
                         </ul>
                     </nav>
-                    <a class="menu_icon">
+                    <a class="menu_icon" style="">
                         <span></span>
                     </a>
                     <a href="cart.html">
-                        <i class="fa">&#xf07a;</i>
+                        <i class="fa" style="padding-right: 20px;">&#xf07a;</i>
                         <span id="nb_article_in_cart"></span>
                     </a>
                     <span id="web_site_name">
-                        <h2>Our website</h2> </span>
+                        <h2>LaxyaCar</h2> </span>
                 </div>
             </header>
             <!--The following div contain a title part (h1) and a div in which the code will create dynamically some
                 article for a random page-->
             <div id="mainContainer">
-                <h1>Tout les pneus</h1>
+                <h1 id="all_the_category">Tout les articles</h1>
                 <div id="div_container_all_article"></div>
             </div>
         </div>
     </body>
-</html>
\ No newline at end of file
+</html>
+<script type="module">
```

Comment:
no JS directly in the HTML + what is this code for ?

Reply:
Not used anymore